ci/minikube/test-conbench-on-mk.sh

```diff
@@ -190,5 +190,24 @@ kubectl logs deployment/conbench-deployment --all-containers > conbench_containe
 cat conbench_container_output.log
 
 # Require access log line confirming that the /metrics endpoint was hit.
-sleep 2
-grep '"GET /metrics HTTP/1.1" 200' conbench_container_output.log
```

Comment:
In local dev I saw that sometimes this failed -- race condition. Not seen in CI yet, but the below version is more robust.